gradle.properties

```diff
@@ -4,7 +4,7 @@ version = 0.0.8-SNAPSHOT
 # issue resolution.
 
 # Internal dependencies:
-mojoRuntimeVersion = 0.13.18
+mojoRuntimeVersion = 0.13.20
```

Comment:
The only change in this PR is the mojo runtime version update.

Reply:
:+1:.